ratpack-manual/src/content/chapters/37-pac4j.md

```diff
@@ -0,0 +1,94 @@
+# Security using pac4j
+
+The `ratpack-pac4j` extension provides authentication support based on [pac4j](https://github.com/pac4j/pac4j).
+
+`pac4j` is a generic authentication engine which is implemented by many frameworks and supports multiple protocols: OAuth, CAS, OpenID (Connect), SAML, Google App Engine and HTTP (form and basic auth).
```

Comment:
I'm not sure to follow you. Should I provide some examples? `pac4j` is implemented in J2E (j2e-pac4j library), by the SSO CAS server (cas-server-support-pac4j module), in Play (play-pac4j library), in Vertx, Undertow, Sparkjava...

 `pac4j` is an authentication/authorization engine: it has many concepts and classes, a lot of common logic, a general algorithm which can be used by many frameworks after some adaptation like a dedicated `WebContext` for example.

Does it make more sense now?